services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -655,35 +722,34 @@ public void processDlqRequestRejected() {
         individualRequest.setErrorLog(errorLog);
         individualRequest.setRequestStatus(IIndividualRequest.IndividualRequestStatus.REJECTED.getStatus());
 
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
-
         final IBulkSubmission bulkSubmission = individualRequest.getBulkSubmission();
 
-        EasyMock.expect(
-                this.mockIndividualRequestDao.queryAsCount(EasyMock.same(IndividualRequest.class), EasyMock.isA(Supplier.class))).andReturn(0L);
+        when(
+                this.mockIndividualRequestDao.queryAsCount(eq(IndividualRequest.class), isA(Supplier.class))).thenReturn(0L);
 
         mockIndividualRequestDao.persist(bulkSubmission);
```

Comment:
This line of code was used to set up expected behaviour under EasyMock whilst it was in "record" mode so it can be checked later on. Under Mockito it's purpose is covered by the following verify() line, so I don't think it is needed.

Reply:
Line is still present (processDlqRequestRejected()).

Reply:
done